tests/test_all.sh

```diff
@@ -31,6 +31,7 @@ echo ""
 # "call" used in test_all_run.sh
 for call in \
   "$GEOIPS_PACKAGES_DIR/geoips/tests/utils/check_code.sh all $repopath" \
+  "pytest --cov $pkgname $GEOIPS_PACKAGES_DIR/geoips/tests" \
```

Comment:
This will be put in check_code.sh (calling both geoips and <pkg>, and not called directly from test_all.sh.

--cov=geoips to the geoips call, --cov=<pkg> for the plugin package call

Reply:
Implemented, please recheck and retest

Reply:
https://github.com/NRLMMD-GEOIPS/geoips/pull/396#discussion_r1381140540

A couple slight changes, looks good!